### PR TITLE
Fixed grammatical errors in feature_request.md

### PR DESCRIPTION
**Pull request by BarbaraStanley**
Reworded the feature_request markdown file

Fixes: #442 

Signed-off-by: Barbara Stanley barbeaboo@gmail.com

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ about: Suggest a new feature for this project
 labels: 'enhancement'
 ---
 
-Features that impact the whole project (e.g. Adding a new OpenJDK distribution) are made over at the [TSC repository](https://github.com/AdoptOpenJDK/TSC/).
+Features that impact the whole project (e.g. Adding a new OpenJDK distribution) are proposed and discussed in the [TSC repository](https://github.com/AdoptOpenJDK/TSC/).
```
Comment (karianna) [inline .github/ISSUE_TEMPLATE/feature_request.md:7]: It's no longer the TSC repo but the [main Adoptium repository](https://github.com/adoptium/adoptium/)

**Review by karianna — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (BarbaraStanley) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I have fixed the link

**Review by karianna — APPROVED** [review]
LGTM

**Review by zdtsw — APPROVED** [review]

**Comment by karianna** [comment]
@BarbaraStanley This will require a rebase to fix the merge conflict.

**Pull request closed without merging**